### PR TITLE
HOMER-487: Style the main menu dropdown

### PR DESCRIPTION
**Pull request by iris-i**

### CODE REVIEW

File: src/components/main-menu/_main-menu.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    > ul {
+      display: none;
+    }
+
+    // Style dropdown buttons
+    button {
+      display: block;
+
+      &::after {
+        background-image: url('../files/icons/menu-dropdown.svg');
```
Comment (jaube05) [inline src/components/main-menu/_main-menu.scss:30]: Can you switch the arrows, currently showing arrow-right then click and arrow-down. Should it not be arrow-down click then arrow-up.

File: src/components/main-menu/main-menu.js [hunk 2]
```diff
@@ -1,21 +1,56 @@
...
+            Company
+          </button>
+          <ul>
+            {/*<li><Link to="/">About</Link></li>*/}
+            <li><Link to="/team">Team</Link></li>
+            <li><Link to="/press">Press</Link></li>
+          </ul>
+        </li>
+        <li className='menu--list__item'><Link to='/services'>Services</Link></li>
+        <li className='menu--list__item'><Link to='/case-studies/'>Our work</Link></li>
```
Comment (jaube05) [inline src/components/main-menu/main-menu.js:42]: Maybe check with Melinda, to see if any other menu links hold child item should show a drop down?

**Review by jaube05 — CHANGES_REQUESTED** [review]
Looks great!

**Comment by iris-i** [comment]
Ok @jaube05, this is ready for another look. New changes:

1. The dropdown icons have been rotated.
2. Extra submenu added under "Our Work"
3. Code to close open submenus when another one is opened.
4. Moved the footer menu code into a different component.

**Review by jaube05 — APPROVED** [review]

**Comment by mikang** [comment]
@iris-i @jaube05 Looks great! Thanks for the quick fix. Could you please make it so that when a user clicks anywhere outside the menu, the menu closes?

**Comment by iris-i** [comment]
Sounds good @mikang, I created https://civicactions-home.atlassian.net/browse/HOMER20-538 for that.

**Comment by mikang** [comment]
> Sounds good @mikang, I created https://civicactions-home.atlassian.net/browse/HOMER20-538 for that.

Thank you, @iris-i!

**Pull request merged**